website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -341,3 +341,80 @@ In the configuration format for the model SQL file:
 </File>
 
 </VersionBlock>
+
+<VersionBlock firstVersion="1.6">
+
+## Dynamic Tables
+
+[Dynamic Tables](https://docs.snowflake.com/en/user-guide/dynamic-tables-about) are Snowflake's flavor of Materialized Views.
+
+### Parameters
+
+dbt-snowflake requires the following parameters:
+
+- `TARGET_LAG`
+- `WAREHOUSE`
+
+To learn more about each parameter and what values it can take, see 
+the Snowflake docs page: [`CREATE DYNAMIC TABLE: Parameters`](https://docs.snowflake.com/en/sql-reference/sql/create-dynamic-table)
```

Comment:
@sfc-gh-slionheart @mikealfare 
how's this for calling out the parameters? I'd rather not have to duplicate y'all's great docs page, but could buy the argument that this doesn't result in the best end-user experience

Reply:
I think I'm fine with that, though I thought the parameter for warehouse was snowflake_warehouse ([here](https://docs.getdbt.com/reference/resource-configs/snowflake-configs#configuring-virtual-warehouses)). Also, the arguments are lowercase, so `target_lag` and `snowflake_warehouse`. They come in as strings that are case-sensitive in python and we don't adjust the case.